### DISCUSSION TITLE
version

### DISCUSSION THREAD

**Discussion by wangqingpen** [edgexfoundry/community#203] [Q&A]
I want to change my version edgex-sdk-go v1 to v2, what are the changes from v1 to v2?

**Comment by MonicaisHer** [comment]
Here is the changelog: https://github.com/edgexfoundry/edgex-go/blob/main/CHANGELOG.md

**Comment by lenny-goodell** [comment]
Here is migration documentation for V2. 
https://docs.edgexfoundry.org/2.3/V2TopLevelMigration/

Please note that V3 is now out and we recommend moving to V3, but you will still need to understand the V1 -> V2 changes before the V2 -> V3 changes.
https://docs.edgexfoundry.org/3.0/V3TopLevelMigration/
